hitch-2011/hitch-fe
# Feature/side navigation

## Is this a fix or a feature?
This is a feature
## What is the change?
* Hamburger Icon has been added to the app
  * This can live on the left or right side of screen
* Menu toggles open and closed between clicks
* Users can travel to different links from menu
* Tests added for user flow.
* Temp image added to the landing screen
## What does it fix?
n/a
## Where should the reviewer start?
* Review the new Menu component and tests
## How should this be tested?
toggle the menu open and close, click links to make sure all functionality is there OR run cypress to automatically check functionality
## Relevant ScreenShots
![Screen Shot 2021-05-17 at 11 02 52 AM](https://user-images.githubusercontent.com/72312636/118534819-e7b4ba80-b706-11eb-91b6-cdbbdf80569a.png)
